Cargo.toml

```diff
@@ -34,7 +34,7 @@ ark-bn254 = "0.4"
 ark-ec = "0.4"
 ark-ed-on-bn254 = "0.4"
 ark-ff = "0.4"
-ark-serialize = "0.4" # features = ["derive"]
+ark-serialize = { workspace = true }
 ark-std = { version = "0.4", default-features = false }
```

Comment:
I'm confused, isn't this file already the workspace's root Cargo.toml?

Reply:
Yes, good catch! I shouldn't have modified this one.